dspace/config/dspace.cfg

```diff
@@ -1429,12 +1252,7 @@ webui.content_disposition_threshold = 8388608
 # If webui.html.max-depth-guess is zero, the request filename and path must
 # always exactly match the bitstream name.  Default value is 3.
 #
-# The 'webui.*' setting is for the JSPUI, and
-# the 'xmlui.*' setting is for the XMLUI
-#
 # webui.html.max-depth-guess = 3
```

Comment:
I could not find any reference to this.

Reply:
@mwoodiupui : I've left this setting in here as I was uncertain whether it'd be useful eventually. If you look a few lines above this, there's a new TODO which says:
`TODO: UNSUPPORTED in DSpace 7.0. May be re-added in a later release`

So, rather than removing it now, I left a TODO here as a note to us to review this later.

Reply:
Good enough.